src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,25 @@
+module Crocodealer.Config
+  ( LabelRule
+  , Label
+  , Package
+  , Config
+  ) where
+
+data Label = Label
+  { name        :: String
+  , description :: String
+  , color       :: String -- Data.Colour.sRGB24read can convert string specifications like "#00aaff" or "00aaff" 
```

Comment:
We definitely should use more typesafe types here.

1. `newtype Name a = Name { unName :: Text }` and `Name Label` for the `labelName` field
2. `Text` for description is okay
3. `newtype HexColor = HexColor { unHexColor :: Text }` for `labelColor`. Constructor of `HexColor` shouldn't not exported. Instead, you need to create smart constructor:

```haskell
mkHexColor :: Text -> Maybe HexColor
```

Which verifies that the given text is in proper format: `#00ffaa` or something like that.